src/spotify.c

```diff
@@ -2042,7 +2044,7 @@ void
 spotify_oauth_callback(struct evbuffer *evbuf, struct evkeyvalq *param, const char *redirect_uri)
 {
   const char *code;
-  const char *err;
+  const char *err = "";
```

Comment:
This was to silence a compiler warning (which actually went away when the called function moved to another file with the merge with master...). It's still a good idea for the called function to init err to avoid random errors if the token calls are changed later, I'll move it in there...